src/js/files/components/UploadTime.tsx

```diff
@@ -0,0 +1,48 @@
+import { formatDuration, intervalToDuration } from "date-fns";
+import React from "react";
+import styled from "styled-components";
+import { getFontSize, getFontWeight } from "../../app/theme";
+
+const StyledUploadInformation = styled.div`
+    display: flex;
+    justify-content: space-between;
+    font-size: ${getFontSize("md")};
+    font-weight: ${getFontWeight("normal")};
+    margin-top: 5px;
+`;
+
+function isDurationZero(duration: Duration): boolean {
+    return Object.values(duration).every(value => value === 0 || value === undefined);
```

Comment:
```suggestion
    return every(duration, (value) => value === 0 || value === undefined);
```
cleaner syntax and no need to iterate the duration twice via lodash

Also is undefined a valid value for members of a `Duration` object? seems like it shouldn't be

Reply:
Since each property of `Duration` is optional, they are implicitly undefined if they are not provided

Reply:
Hmm, the tests and prettier seem to fail after implementing this change - was there an import you added for this?